apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -539,7 +548,8 @@ export class ChangeAction extends BaseRushAction {
    * Detects the user's email address from their Git configuration, prompts the user to approve the
    * detected email. It returns undefined if it cannot be detected.
    */
-  private async _detectAndConfirmEmail(promptModule: inquirer.PromptModule): Promise<string | undefined> {
+  // eslint-disable-next-line @typescript-eslint/typedef
+  private async _detectAndConfirmEmail(promptModule): Promise<string | undefined> {
```

Comment:
As I explained with `IYarnLockfileParseResult`, just use a regular `import` with a TODO comment saying to convert to  "import type" later.  *Don't remove our type annotations.*